test/move-file.js

```diff
@@ -162,11 +170,27 @@ t.test('move a directory across devices', async t => {
     'created the symlink with the appropriate target'
   )
   t.ok(fsSync.lstatSync(`${dest}/abs`).isSymbolicLink(), 'created the absolute path symlink')
-  t.equal(
-    fsSync.readlinkSync(`${dest}/abs`),
-    process.platform === 'win32' ? `${process.cwd()}\\` : process.cwd(),
-    'kept the correct absolute path'
-  )
+  if (process.platform === 'win32') {
+    if (hasTrailingSlashFix) {
```

Comment:
This seems a little overly verbose.  We have only two paths here from what I can tell "win32 and doesn't have fix" and "everything else"

It'd also probably be cleaner to conditionally set up the value we're comparing against and run one `t.equal`